idd/Energy+.idd.in

```diff
@@ -57999,7 +57542,7 @@ EvaporativeCooler:Indirect:ResearchSpecial,
        \note x = HXFlowRatio = sum of the primary and secondary flow rates divided by the sum of the design flow
        \note rates. If this input field is left blank, constant cooler wetbulb effectiveness is assumed.
        \type object-list
-       \object-list AllCurves
+       \object-list OneDimCurves
        \note Any curve or table with one independent variable can be used:
        \note Curve:Linear, Curve:Quadratic, Curve:Cubic, Curve:Quartic, Curve:Exponent,
        \note Curve:ExponentialSkewNormal, Curve:Sigmoid, Curve:RectuangularHyperbola1,
```

Comment:
And then we have these notes that attempt to list any one-dimensional curve type - these could be cleaned out.  Hmmm, does the epJSON schema even preserve any of this information, currently? Should it? Could it?